public/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Freedomvote</title>
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+    <header class="navigation">
+      <nav class="navigation__nav">
+        <a href="#">
+          <img class="navigation__nav__logo" src="/img/logo.svg" />
+        </a>
+        <ul class="navigation__nav__page">
+          <li><a href="#" class="navigation__nav__page__active">Home</a></li>
+          <li><a href="#">Kandidaten</a></li>
+          <li><a href="#">Vergleichen</a></li>
+          <li><a href="#">Partner</a></li>
+        </ul>
+        <i class="navigation__nav__divider"></i>
+        <nav class="navigation__nav__language">
+          <ul>
+            <li>DE</li>
+            <li>IT</li>
+            <li>NL</li>
+            <li>EN</li>
+            <li>FR</li>
+          </ul>
+        </nav>
+        <i class="navigation__nav__search">Search</i>
+      </nav>
+    </header>
+    <div class="container">
+      <h1 class="hero">Freiheit und Sicherheit war noch nie einfach</h1>
+      <h2 class="headline">Freiheit und Sicherheit war noch nie einfach</h1>
+      <h2 class="headline-fancy">Freiheit und Sicherheit war noch nie einfach</h1>
+
+      <p>Besonders jetzt im Informationszeitalter wird das Thema immer schwieriger und komplexer. Freiheit ermöglicht uns einen innovativen Umgang mit Technologie, schafft Zugang zu Wissen und fördert Kreativität. Der Staat ist jedoch auch in der Verantwortung für Sicherheit zu sorgen, damit diejenigen zur Rechenschaft gezogen werden können, welche die ermöglichten Freiheiten missbrauchen.</p>
+      <div class="boxes">
```

Comment:
Creeate a BEM Component. For Example:
`<div class="box"><div class="box__item"></div></div>"`

My naming is bad… im sure you can find a better name :P